recommender/recommender.py

```diff
@@ -999,11 +1008,11 @@ def parse_xml(cls, node, runtime, keys, _id_generator):  # pylint: disable=unuse
 
         """
         block = runtime.construct_xblock_from_class(cls, keys)
-        lines = []
-        for line in node.text.split('\n'):
-            line = line.strip()
-            if len(line) > 2:
-                lines.append(json.loads(line))
+        lines = json.loads(node.text)
```

Comment:
I believe this will fail if there is no default. If we have, for example: 
    <recommender />

An example like this should be included as a scenario so we are testing it.